Tests/Unit/Domain/Dto/ImportJobTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SUDHAUS7\Xlsimport\Tests\Unit\Domain\Dto;
+
+use SUDHAUS7\Xlsimport\Domain\Dto\ImportJob;
+use TYPO3\TestingFramework\Core\Unit\UnitTestCase;
+
+final class ImportJobTest extends UnitTestCase
+{
+    /**
+     * @test
+     */
+    public function createdDtoHasCorrectValues(): void
+    {
+        $table = 'test_table';
```

Comment:
These values are used for assertions lateron, as a best practice normally these things are prefixed with `expected`

```php
$expectedTable = 'test_table';
$expectedJsonFile = 'test.json';
// ...
```

Reply:
yes :)